storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
+Triggers can catch certain events and execute basic commands on your sprite in response.
+
+.. note:: Triggers are the most complex command currently provided by the storyboarding API and unfortunately they are accordingly buggy.
+  There is a whole batch of weird behaviour attached to the use of triggers that will be covered in its most common oddities throughout this chapter and at the end of this chapter in its less common oddities.
+
+
+Syntax
+------
+Like every other command, a trigger is always attached to a sprite or animation.
+
+Triggers consist of merely 4 elements:
+    - command-character
+    - trigger condition
+    - time window
+    - trigger group (optional)
+
+After the trigger command you have to list the commands that are to be executed when the trigger condition occurs.
+
+.. code-block:: c
+  :linenos:
+  :caption: An example of using a simple trigger.
+  :name: TriggerSyntax
+
+  Sprite,Background,Centre,"sb\aoba.png",192.625,140
+   F,0,23500,,0
+   S,0,23500,,0.2
+   T,HitSoundSoft,0,105330
+    F,0,0,,1
+    F,0,110,,0
+
+*T* is they character marking the triggercommand.
+
+*HitSoundSoft* is the trigger condition. You will learn about the different kinds of trigger conditions in the next chapter.
+
+*0,105330* marks start- and endtime of the trigger. There is no limit to how often a trigger can be executed within the specified timeframe.
+
+
+| The Fade commands after the trigger command are indented by one additional whitespace or underscore to mark them as belonging to the trigger.
+| It is very important to notice that the time values specified for these commands are **relative** to the time when the trigger starts executing.
+
+| Example: If the trigger condition is met at 23918 the sprite will change to 100% opacity at 23918 and fade out at 24028.
+| This behaviour assures that there a trigger can be activated and executed multiple times.
+
+Trigger Conditions
+------------------
+| Triggering a trigger is very similar to triggering a trap:
+| It will snap once someone steps into it.
+
+| It is not *that* simple though. Our traps have different tastes and they are picky to different degrees.
+| Let's check how we can satisfy the taste of each of our traps.
+
+Hitsounds
+~~~~~~~~~
+| The first big chunk of possible trigger conditions consists of hitsounds. Every time a note is hit, a hitsound will be played. But Hitsound does not equal Hitsound:
+| There are as many possible different triggers for hitsounds in a map as hitsounds available.
+
+| How do we specify the hitsound we want to have our trap snap on?
+| This is easy. The naming for the according trigger follows a simple naming scheme as shown in the following table:
+
+.. rst-class:: table table-sm table-hover table-striped
+.. csv-table:: Hitsound Trigger Conditions
+   :header: "Condition Type", "Name of Sampleset", "Additional", "Number of Sampleset", "Triggername"
```

Comment:
Changing to "Sampleset" would be ambiguous as it remains unclear whether it addresses soft/normal/drum or 1,2,3,4 without looking at the context. 
I will change that to "Sampleset Name".

As for the Additional, this derives from the editor's header for the per-note sampleset dropdown on the top left (reads "Additions"), it makes sense to change this to "Hitsound Addition". Only "Hitsound" is too inaccurate of a term to use because it is not possible to specify hitnormals as an own type of hitsound here.

Reply:
Could we opt for Hitsound Addition, then?

Reply:
Yes, done.